buildchow/templates/product-template.yaml

```diff
@@ -117,39 +109,44 @@
             - maven.test.error.ignore=true
             - download.cache.directory={download-cache}
             - targetPlatformGroup=jbdevstudio
-            - jbosstools-site=http://download.jboss.org/jbosstools/updates/nightly/core/${{jbosstools_site_stream}}/
+            - jbosstools-site=http://download.jboss.org/jbosstools/mars/snapshots/builds/jbosstools-build-sites.aggregate.site_${{jbosstools_site_stream}}/latest/all/repo/
           pom: ${{WORKSPACE}}/sources/product/site/pom.xml          
       - maven-target:
-          goals: "org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
-          maven-version: maven-3.x
+          goals: "-U org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
```

Comment:
ah, yeah - I guess if we continue to use releng snapshot then it is necessary.

But:

a) shouldn't releng be released before codefreeze so we stop using snapshots for our builds ? (that is one of the reasons we wanted to get away from just using latest master version of releng scripts)

b) higher up we have:
       `goals: "{ci-maven-flags} org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack" #TODO: should it have all MAVEN_FLAGS ?`
  so it does the same thing; but here it is usign ci-maven-flags. They should at least use the same approach.